phpBB/includes/functions_user.php

```diff
@@ -424,11 +424,11 @@ function user_add($user_row, $cp_data = false, $notifications_data = null)
 }
 
 /**
- * Remove User
+ * Delete User(s) and all related.
  *
- * @param string	$mode		Either 'retain' or 'remove'
- * @param mixed		$user_ids	Either an array of integers or an integer
- * @param bool		$retain_username
+ * @param string	$mode				Mode (retain|delete)
```

Comment:
It might be better to use the way you described it in the events below, which is more descriptive:
> Mode of posts deletion (retain|delete)